Makefile

```diff
@@ -94,3 +85,5 @@ bumpversion-to-dev:
 bumpversion-from-dev:
 	tox -e bumpversion-from-dev
 
+vendor-sweeps:  # expects an argument called ref=,  example usage: make vendor-sweeps ref=master
+	bash tools/vendor_sweeps.sh $(ref)
```

Comment:
This makefile target clones the sweeps repository to a temporary directory, checks out the ref pointed to by $ref, then copies over the requirements.txt file (to install the sweeps extras in wandb/client) and the library contents to wandb/sweeps